packages/paste-core/components/badge/src/types.ts

```diff
@@ -5,22 +5,63 @@ import type { BadgeVariants } from "./constants";
 
 export type BadgeSizes = "default" | "small";
 
-export type BadgeBaseProps = Pick<BoxProps, "element"> & {
+export type BadgeBaseProps = {
```

Comment:
suggestion: you might be able to get away with just exporting BadgeProps from index as it documents all the Base, Span and Button props in one. The page kinda feels confusing